### PR TITLE
Capture: Add Element.hasPointerCapture

### PR DESCRIPTION
**Pull request by bethge**
Adds `Element.hasPointerCapture()` (#333)

It seemed pretty straight forward.

### CODE REVIEW

File: src/capture.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ if (n.msPointerEnabled) {
     dispatcher.releaseCapture(pointerId, this);
   };
 }
+h = function hasPointerCapture(pointerId) {
+  return !!dispatcher.captureInfo[pointerId];
+};
```
Comment (scottgonzalez) [inline src/capture.js]: Won't this always return false when `msPointerEnabled` is `true`?

Comment (bethge) [inline src/capture.js]: Ah, of course! Thank you for catching that. Not quite as straight forward as I had hoped.

**Comment by bethge** [comment]
Added Scott's approach to include IE10 support. 

There is one minor caveat for IE10: An explicit `releasePointerCapture()` will directly cause `hasPointerCapture()` to return `false`, but it takes some `pointermove` for IE10 to trigger the actual `lostpointercapture` event which officially ends the capture.
I.e.: _If_ a user does not move his pointer, a `releasePointerCapture()` will cause any `hasPointerCapture()` to return `false` before the `lostpointercapture` event triggers.

**Comment by scottgonzalez** [comment]
I believe that is actually the intended behavior.

[`hasPointerCapture()`](https://w3c.github.io/pointerevents/#dom-element-haspointercapture) is defined as "returns true if the pending pointer capture target override for pointerId is set to the element on which this method is invoked, and false otherwise." The last step of calling [`releasePointerCapture()`](https://w3c.github.io/pointerevents/#h-releasing-pointer-capture) is "For the specified pointerId, clear the pending pointer capture target override, if set."

File: src/capture.js [hunk 2]
```diff
@@ -10,7 +10,11 @@ function assertActive(id) {
   }
 }
 function assertConnected(elem) {
-  if (!elem.ownerDocument.contains(elem)) {
```
Comment (scottgonzalez) [inline src/capture.js]: Why did this need to change?

Comment (bethge) [inline src/capture.js]: I noticed during testing that IE10 does not support `document.contains()`. We ran into this bug once before: https://github.com/jquery/PEP/issues/320

A separate issue would probably have been more appropriate.

Comment (scottgonzalez) [inline src/capture.js]: Ah, ok.

**Pull request closed without merging**